### DISCUSSION TITLE
无法隐藏时间

### DISCUSSION THREAD

**Discussion by xiaobaibukeaiya** [tomakino/CSLyric#38] [General]
澎湃1系统wi无法隐藏状态栏时间

**Comment by Huang2326** [comment]
我也是澎湃1，使用的时候先排除法，把西米露，米客等可以影响状态栏的工具给关闭了，然后如图，时间的显示控件在视图控制器里面是这一项，选择播放时隐藏即可。![Screenshot_20251109-183135.png](https://github.com/user-attachments/assets/dbd5a346-f6cf-4834-abfe-aa30f843d8a5)
